tests/protractor/page-objects/forms/delivery-report.po.js

```diff
@@ -1144,20 +1114,22 @@ module.exports = {
       })
       .then(done, done);
   },
+
   goNext: () => {
```

Comment:
This appears to go to the next page of delivery reports.  A clearer name than:
```
deliveryReport.goNext();
```
might be
```
deliveryReportList.nextPage();
```
Unless we really are going to the next "delivery report", in which case there is still room for improvement e.g. `goToNext()`.